atd-vzd/README.md

```diff
@@ -1,51 +1,70 @@
 ## Vision Zero Database
 
-Currently the Vision Zero database consists of a Postgres 10 database that mirrors the CRIS public crash data format, with added field lookup tables.
+Currently the Vision Zero database consists of a Postgres 14 database that mirrors the CRIS public crash data format with added field lookup tables.
 
-At the moment, only the schema, views and triggers are posted on here and version-controlled. The actual data can be gathered from the [CRIS website]([https://cris.dot.state.tx.us](https://cris.dot.state.tx.us/)). 
+CRIS data can be gathered from the [CRIS website]([https://cris.dot.state.tx.us](https://cris.dot.state.tx.us/)). 
 
-Currently the schema is being tracked in the following folders:
+## Backups
+
+Currently there are two systems making backups, one in RDS and the other in S3 via CRON jobs for 60 days. The backups in S3 are table-based and contain both schema and data for each individual table. 
```

Comment:
I need to check in on this cron job described here. Not sure if we do that anymore.

Reply:
thats a good question, we have this issue open that i keep meaning to find time to discuss
https://github.com/cityofaustin/atd-data-tech/issues/10276

Reply:
Cool - thanks for the background, Frank! I was thinking that this was describing an outdated process. I'll update the language here to reflect what is currently going on.